application/src/main/java/org/togetherjava/tjbot/commands/filesharing/FileSharingMessageListener.java

```diff
@@ -126,13 +136,14 @@ private void processAttachments(@NotNull MessageReceivedEvent event,
         try {
             body = JSON.writeValueAsString(jsonRequest);
         } catch (JsonProcessingException e) {
-            throw new IllegalStateException("Couldn't parse json request!", e);
+            throw new IllegalStateException(
+                    "Attempting to upload a file to gist, but unable to create the JSON request.",
+                    e);
         }
 
         HttpRequest request = HttpRequest.newBuilder()
             .uri(URI.create(SHARE_API))
-            .header("Accept", "application/json")
-            .header("Authorization", "token " + gistApiKey)
+            .headers("Accept", "application/json", "Authorization", "token " + gistApiKey)
```

Comment:
is that auto alignment from spotless? in that case the previous version was better, id revert back.

Reply:
yea I think spotless aligned it differently this time

Reply:
how can I get the old alignment?